pylon/aws/s3.py

```diff
@@ -140,8 +140,8 @@ def put(
 
         return getPath(self.bucket.name, key)
 
-    def get(self, key, encoding=None) -> typing.Tuple[_STR_OR_BYTES, dict]:
```

Comment:
Delete line 149 `metadata = response['Metadata']` and line 152 `logging.debug(f'Metadata: {metadata}')`.